packages/components/src/components/Box/__tests__/Box.ct.tsx

```diff
@@ -0,0 +1,67 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { expect, test } from '@playwright/experimental-ct-react';
+
+import { Box } from '../Box';
+
+const BOX_TEXT = 'sample box';
+
+test('should render without error', async ({ mount }) => {
+    const component = await mount(
+        <Box>
+            <div style={{ border: '1px solid black' }}>{BOX_TEXT}</div>
+        </Box>,
+    );
+    await expect(component).toBeVisible();
+    await expect(component).toContainText(BOX_TEXT);
+    await expect(component).toHaveScreenshot();
```

Comment:
why are we now starting with screenshot tests instead of using `toHaveCSS`?

Reply:
Replaced